index.html

```diff
@@ -148,7 +148,7 @@
 			<div class="sub-description-sub-heading">
 				Ideation
 			</div>
-			<hr class="yellow-accent-line">
+			<hr class="yellow-accent-line-left-sub">
 			<p>We are building a <span id="home-description-emphasis-words">biological band-pass filter</span>. While biological band pass filters have been developed before, none respond to frequency-based inputs but instead respond to amplitude based inputs. By controlling the rates of degradation of the components of our system, we are putting together a <span id="home-description-emphasis-words">frequency-response</span> low-pass and high-pass filter to create a <span id="home-description-emphasis-words">novel</span> band-pass filter.</p>
 			</div>
 			<div class="home-sub-description-left-image">
```

Comment:
Ideally, we'll have at least three or four sub-descriptions. Can you add two more? Also, you might run in the issue I was talking about yesterday with the CSS grid stuff :( If you do, try to do what I mentioned yesterday.